SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpEventParticipantDto.cs

```diff
@@ -103,5 +103,11 @@ public class MpEventParticipantDto
 
         [JsonProperty("End_Date")]
         public DateTime? EndDate { get; set; }
+
+        [JsonProperty("Guest_Sign_In")]
+        public bool GuestSignIn { get; set; }
```

Comment:
This is a new column (added by https://github.com/crdschurch/crds-signin-checkin/pull/411/files) that is nullable.  Will deserialization from json -> Dto break if this column is null?